apps/dashboard/src/components/workflow-editor/test-workflow/test-workflow-form.tsx

```diff
@@ -120,11 +120,11 @@ export const TestWorkflowForm = ({ workflow }: { workflow?: WorkflowResponseDto
             onValueChange={(value) => setActiveSnippetTab(value as SnippetLanguage)}
           >
             <TabsList className="border-t-0" variant="regular">
-              {/* {workflow?.origin === WorkflowOriginEnum.EXTERNAL && (
+              {workflow?.origin === WorkflowOriginEnum.EXTERNAL && (
                 <TabsTrigger className={tabsTriggerClassName} value="framework" variant="regular">
                   Framework
                 </TabsTrigger>
-              )} */}
+              )}
```

Comment:
I'm not sure about showing the code tab here, as it's going to be confusing for the user to see irrelevant code in this tab 🤷 I suggest just remove it altogether

Reply:
Ok, I saw it was commented out thought to be a mistake 😅 
i will remove that part totally. Thanks